lib/rules/require-parentheses-around-arrow-param.js

```diff
@@ -48,6 +48,9 @@ module.exports.prototype = {
     check: function(file, errors) {
         function isWrapped(node) {
             var openParensToken = file.getPrevToken(file.getFirstNodeToken(node));
+            if (node.type !== 'RestElement') {
+                openParensToken = file.getPrevToken(openParensToken);
```

Comment:
Yeah me too - had to console.log some stuff - Kind of wish there was a utility that showed the ast and then allowed logging/programming around it very easily (so it's easy to debug jscs) - like prevToken, nextToken etc.

But yeah it doesn't make sense. I mean't to check for `...`

Doing a PR now - also adding disallow rule since I forgot.